telethon/client/users.py

```diff
@@ -86,7 +87,8 @@ class UserMethods(TelegramBaseClient):
                     raise
                 await self._switch_dc(e.new_dc)
 
-        raise ValueError('Number of retries reached 0')
+        raise ValueError('Request was unsuccessful {} times'
+                        .format(attempt))
```

Comment:
Being a bit picky here but:
```suggestion
                         .format(attempt))
```

To align it like any linter would.